src/PSR7/SpecFinder.php

```diff
@@ -284,4 +300,23 @@ public function findCookieSpecs(OperationAddress $addr) : array
 
         return $cookieSpecs;
     }
+
+    /**
+     * @throws NoCallback
+     */
+    private function findCallbackInOperation(CallbackAddress $addr, Operation $operation) : Operation
+    {
+        $callbacks = $operation->callbacks;
+        if (! isset($callbacks[$addr->callbackName()])) {
+            throw NoCallback::fromCallbackPath($addr->path(), $addr->method(), $addr->callbackName(), $addr->callbackMethod());
```

Comment:
I hope we can pass the whole `addr` here instead of destructing it to args?

Reply:
Yes, we could do that. It was so for all other exceptions, so I thought that there should be no dependency to the address.